### PR TITLE
Serve static directories only when serving Storybook locally

### PR DESCRIPTION
**Pull request by bencodeorg**
We temporarily commented out code that includes static assets when running Storybook locally in order to get our public site back up and running. This change reinstates those directories when running Storybook locally, but continues to exclude them when deploying Storybook.

I have a [separate PR here](https://github.com/code-dot-org/code-dot-org/pull/53569) that gives an example of how we could become less dependent on these static directories when running Storybook, but in the meantime didn't want to regress any functionality if people are running storybook locally and want to access assets as they have before.

I also remove one of the static directories (pegasus), which was [added in this PR](https://github.com/code-dot-org/code-dot-org/pull/38367). The functionality in that PR was [removed](https://github.com/code-dot-org/code-dot-org/pull/49650) in January of this year.

## Testing story

Tested manually that when running storybook locally (ie, via `yarn storybook`), I had access to the assets at the two static directories listed. Also tested that building storybook (ie, via `yarn deploy:storybook` with the github push commented out from `deploy-storybook.sh`) excluded these directories (desired behavior).

### CODE REVIEW

File: apps/.storybook/main.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 module.exports = {
   stories: ['../src/**/*.story.@(js|jsx|ts|tsx)'],
-  staticDirs: [
-    // Temporarily commented out to test whether we can deploy storybook to our public repo (see storybook-deploy.sh for details)
-    // In the meantime, feel free to uncomment these if you need them for serving storybook locally
-    // '../build/package/',
-    // '../../dashboard/public',
-    // '../../pegasus/sites.v3/code.org/public'
-  ],
+  staticDirs: staticDirs,
```
Comment (fisher-alice) [inline apps/.storybook/main.js]: nit: could use `staticDirs,` instead of `staticDirs: staticDirs,`

File: apps/.storybook/preview-head.html [hunk 2]
```diff
@@ -3,7 +3,6 @@
 <link rel="stylesheet" media="all" href="css/levelbuilder.css" />
 <link rel="stylesheet" media="all" href="css/pd.css" />
 <link rel="stylesheet" media="all" href="css/droplet/droplet.min.css" />
-<link rel="stylesheet" media="all" href="/css/brstrap.css" />
```
Comment (fisher-alice) [inline apps/.storybook/preview-head.html:6]: Why is this no longer needed?

File: apps/envConstants.js [hunk 3]
```diff
@@ -45,5 +45,7 @@ module.exports = {
   BROWSER: process.env.BROWSER,
   // If set, will turn on react hot loader and run the webpack dev server
   HOT: !!process.env.HOT,
-  LEVEL_TYPE: process.env.LEVEL_TYPE
+  LEVEL_TYPE: process.env.LEVEL_TYPE,
+  // Include static assets when serving storybook locally
+  STORYBOOK_STATIC_ASSETS: process.env.STORYBOOK_STATIC_ASSETS,
```
Comment (fisher-alice) [inline apps/envConstants.js:50]: Very new to this - would you mind explaining when/how `process.env.STORYBOOK_STATIC_ASSETS` is assigned?

**Review by fisher-alice — COMMENTED** [review]
Checked out branch and tested locally 🎉 ! Very new to this area so asking a couple questions.

File: apps/.storybook/preview-head.html (hunk 2 above)
Comment (bencodeorg) [reply apps/.storybook/preview-head.html:6]: It's related to the removal of `../../pegasus/sites.v3/code.org/public` as one of the static directories, there are a couple of links in the description to relevant PRs that have more detail. tl;dr is that I think it was added to support a component in Storybook that is no longer in Storybook.

File: apps/envConstants.js (hunk 3 above)
Comment (bencodeorg) [reply apps/envConstants.js:50]: mmm yeah, when we run `npm run storybook`, I prefix starting storybook by setting an env variable `STORYBOOK_STATIC_ASSETS=1`.

Then, you can access your environment variables via `process.env` in Node:

https://nodejs.dev/en/api/v18/process/#processenv

Comment (fisher-alice) [reply apps/envConstants.js:50]: Thanks!

**Review by fisher-alice — APPROVED** [review]
LGTM!

**Pull request merged**